assets/css/src/generic/_forms-checkbox.scss

```diff
@@ -0,0 +1,38 @@
+.mailpoet-form-checkbox {
+  align-items: center;
+  cursor: pointer;
+  display: inline-flex;
+  position: relative;
+  vertical-align: middle;
+
+  input {
+    height: 1px;
+    position: absolute;
+    visibility: hidden;
+    width: 1px;
+  }
+
+  &.mailpoet-full-width + .mailpoet-form-checkbox.mailpoet-full-width { margin-top: $grid-gap; }
+  &:not(.mailpoet-full-width) + .mailpoet-form-checkbox:not(.mailpoet-full-width) { margin-left: $grid-gap; }
+}
+
+.mailpoet-form-checkbox-control {
+  align-items: center;
+  border: 1px solid $color-input-border;
+  border-radius: 4px;
+  box-sizing: border-box;
```

Comment:
It would be nice, but it would break a lot of things (everything with padding or border and specific width/height would become smaller). So we have to do it component by component and maybe when the redesign is all done, it can be set to all elements. But I still wouldn't count on that.